pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/AccessorMethodGenerationRule.java

```diff
@@ -81,24 +80,24 @@ static void checkMemberAccess(RuleContext ruleContext, JavaNode refExpr, JAccess
             }
             assert node != null : "Node should be in the same compilation unit";
             if (reportedNodes.add(node)) {
-                ruleContext.addViolation(node, stripPackageName(refExpr.getEnclosingType().getSymbol()));
+                ruleContext.addViolation(node, canonicalNameWithoutPackage(refExpr.getEnclosingType().getSymbol()));
             }
         }
     }
 
     /**
-     * Returns the canonical name without the package name. Eg for a
-     * canonical name {@code com.github.Outer.Inner}, returns {@code Outer.Inner}.
+     * Returns the canonical name without the package name.
+     * Eg for a canonical name {@code com.github.Outer.Inner}, returns {@code Outer.Inner}.
      */
-    private static String stripPackageName(JClassSymbol symbol) {
```

Comment:
stripPackageName is kind of broad canonicalNameWithoutPackage is way more specific. So its 3 words vs 4 and smaller scopes methods should be very precice.